lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,38 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: ['derpiboo.ru'],
+	matchRe: /^https?:\/\/(?:www\.)?derpiboo.ru\/\d+$/i,
```

Comment:
Yeah, that's starting to get a bit complicated for a Regex. Certainly still doable, but it might be cleaner if I used a real URL parser. Is https://developer.mozilla.org/en-US/docs/Web/API/URL an option, or does RES need to support older browsers?